ui/jobs/combo_tracker.ts

```diff
@@ -1,7 +1,19 @@
 import { kAbility, kComboBreakers } from './constants';
 
+type StartMap = Record<string, {
+  id?: string;
+  next?: StartMap;
+}>;
+
 export default class ComboTracker {
-  constructor(comboBreakers, callback) {
+  comboTimer?: number | undefined;
+  comboBreakers: readonly string[];
+  startMap: StartMap;
+  callback: (id: string | null) => void;
+  considerNext: StartMap;
+  isFinalSkill: boolean;
+
+  constructor(comboBreakers: readonly string[], callback: (id: string | null) => void) {
```

Comment:
Would it be worth having a `type ComboSkill = string | null` to encapsulate this here and elsewhere? Maybe also a `type ComboTrackerCallback = (id: ComboSkill) => void;`?